spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/core/UserAgentHeaderProviderIntegrationTests.java

```diff
@@ -39,7 +40,7 @@
 	public void testGetHeaders() {
 		UserAgentHeaderProvider subject = new UserAgentHeaderProvider(this.getClass());
 
-		String versionRegex = "\\d+\\.\\d+\\.\\d+\\.[BUILD-SNAPSHOT|M\\d+|RC\\d+|RELEASE]$";
+		String versionRegex = "\\d+\\.\\d+\\.\\d+(\\-SNAPSHOT)?";
```

Comment:
I imagine you'll want milestone releases too with the `-M1` suffix ?

Reply:
I thought we decided not to do milestone releases.